cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -168,6 +168,8 @@ typedef enum {
     UART_FLAG_NONE            = 0x0,    /**< No flags set */
     UART_FLAG_RUN_STANDBY     = 0x1,    /**< run SERCOM in standby mode */
     UART_FLAG_WAKEUP          = 0x2,    /**< wake from sleep on receive */
+    UART_FLAG_RXINV           = 0x4,    /**< invert RX signal */
+    UART_FLAG_TXINV           = 0x8,    /**< invert TX signal */
```

Comment:
Does it make sense to jsut invert one of them?

Reply:
Id it doesn't it could be a single flag.

Reply:
> Does it make sense to jsut invert one of them?

We have some possible optimization to the [DOSE](https://doc.riot-os.org/group__drivers__dose.html) bus where a single MOSFET is used instead of a CAN transceiver. But this will invert the TX signal:


![image](https://user-images.githubusercontent.com/1301112/84881498-39376e00-b08e-11ea-8192-08b7f77a903f.png)
</details>

Reply:
Ok!

Reply:
Meh, this does not what I expected it to do.
It will only invert the individual bits of the byte sent over UART, but the UART level (idle HIGH) will remain the same.